external/vcm/vcm/derived_mapping.py

```diff
@@ -360,3 +360,43 @@ def water_vapor_path(self):
     return da.assign_attrs(
         {"long_name": "column integrated water vapor", "units": "mm"}
     )
+
+
+@DerivedMapping.register(
```

Comment:
Are these only used for your specific model in`fv3fit`? If so, I would consider isolating these near your fv3fit code. You can call `DerivedMapping.register` from any file...doesn't have to be here.

Reply:
Good idea.

Reply:
Hmm realizing this won't actually work, since I want the data provided by `loaders` to include these derived quantities. Other than the possible usage of the fine-res temperature nudging tendency (which is not required) they are generally useful.

Reply:
I left the `upward_heat_flux_at_surface` derived quantity here since it is general (even if not currently used elsewhere) and moved the TOA net flux calculation to the fine-res loader.